sql/expression/function/convert_tz_test.go

```diff
@@ -125,10 +125,20 @@ func TestConvertTz(t *testing.T) {
 			toTimeZone:     "10:00",
 			expectedResult: nil,
 		},
+		{
+			name:           "Test with @@GLOBAL.time_zone value: SYSTEM",
+			datetime:       time.Date(2010, 6, 3, 12, 12, 12, 0, time.UTC),
+			fromTimeZone:   "SYSTEM",
+			toTimeZone:     "+01:00",
+			expectedResult: time.Date(2010, 6, 3, 13, 12, 12, 0, time.UTC),
+		},
```

Comment:
Would be good to cover both fromTimeZone and toTimeZone just to make sure both of those codepaths are being tested. 

```suggestion
		{
			name:           "Test fromTimeZone value: SYSTEM",
			datetime:       time.Date(2010, 6, 3, 12, 12, 12, 0, time.UTC),
			fromTimeZone:   "SYSTEM",
			toTimeZone:     "+01:00",
			expectedResult: time.Date(2010, 6, 3, 13, 12, 12, 0, time.UTC),
		},
		{
			name:           "Test toTimeZone value: SYSTEM",
			datetime:       time.Date(2010, 6, 3, 12, 12, 12, 0, time.UTC),
			fromTimeZone:   "+01:00",
			toTimeZone:     "SYSTEM",
			expectedResult: time.Date(2010, 6, 3, 11, 12, 12, 0, time.UTC),
		},
```